extensions/binding/org.eclipse.smarthome.binding.tradfri/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,123 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="tradfri"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+        xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+    <bridge-type id="gateway">
+        <label>Trådfri Gateway</label>
+
+        <config-description>
+            <parameter name="host" type="text" required="true">
+                <context>network-address</context>
+                <label>Host</label>
+                <description>Hostname or IP address of the IKEA Trådfri gateway</description>
+            </parameter>
+            <parameter name="port" type="integer" required="false">
+                <label>Port</label>
+                <description>Port for accessing the gateway</description>
+                <advanced>true</advanced>
+                <default>5684</default>
```

Comment:
How does the default configuration work is based on this or the one in the class. If by accident they were different which one would win? Should you add the one in the class?